nautobot_ssot_infoblox/jobs.py

```diff
@@ -29,44 +34,40 @@ class Meta:  # pylint: disable=too-few-public-methods
 
     @classmethod
     def data_mappings(cls):
-        """Shows mapping of models between Infoblox and Nautobot."""
+        """Show mapping of models between Infoblox and Nautobot."""
         return (
             DataMapping("network", None, "Prefix", reverse("ipam:prefix_list")),
             DataMapping("ipaddress", None, "IP Address", reverse("ipam:ipaddress_list")),
-            # DataMapping("vlan", None, "VLAN", reverse("ipam:vlan_list")),
-            # DataMapping("vlanview", None, "VLANGroup", reverse("ipam:vlangroup_list")),
+            DataMapping("vlan", None, "VLAN", reverse("ipam:vlan_list")),
+            DataMapping("vlanview", None, "VLANGroup", reverse("ipam:vlangroup_list")),
         )
 
-    def sync_data(self):
-        """Method to handle synchronization of data to Nautobot."""
+    def load_source_adapter(self):
+        """Load Infoblox data."""
         self.log_info(message="Connecting to Infoblox")
-        infoblox_adapter = infoblox.InfobloxAdapter(job=self, sync=self.sync)
+        client = InfobloxApi()
+        self.source_adapter = infoblox.InfobloxAdapter(job=self, sync=self.sync, conn=client)
         self.log_info(message="Loading data from Infoblox...")
-        infoblox_adapter.load()
+        self.source_adapter.load()
+
+    def load_target_adapter(self):
+        """Load Nautobot data."""
         self.log_info(message="Connecting to Nautobot...")
-        nb_adapter = nautobot.NautobotAdapter(job=self, sync=self.sync)
+        self.target_adapter = nautobot.NautobotAdapter(job=self, sync=self.sync)
         self.log_info(message="Loading data from Nautobot...")
-        nb_adapter.load()
-        self.log_info(message="Performing diff of data between Infoblox and Nautobot.")
-        flags = DiffSyncFlags.CONTINUE_ON_FAILURE | DiffSyncFlags.SKIP_UNMATCHED_DST
-        diff = nb_adapter.diff_from(infoblox_adapter, flags=flags)
```

Comment:
Is this where some of the performance changes are happening? I am curious why this and the below lines were removed?

Reply:
This is part of the changes to use the load adapter methods to gather performance metrics. This was done as a precursor to the performance changes so I could gather statistics. You can see that the flags are now set in the under init method instead. The diff from is called automatically once the adapter load methods are called.